test/e2e/storage/vsphere/vsphere_scale.go

```diff
@@ -78,27 +75,22 @@ var _ = utils.SIGDescribe("vcp at scale [Feature:vsphere] ", func() {
 		nodeVolumeMapChan = make(chan map[string][]string)
 
 		// Read the environment variables
-		volumeCountStr := os.Getenv("VCP_SCALE_VOLUME_COUNT")
-		Expect(volumeCountStr).NotTo(BeEmpty(), "ENV VCP_SCALE_VOLUME_COUNT is not set")
-		volumeCount, err = strconv.Atoi(volumeCountStr)
+		Expect(VCPScaleVolumeCount).NotTo(BeEmpty(), "ENV VCP_SCALE_VOLUME_COUNT is not set")
```

Comment:
The name of the env var now appears in 2 different files. It might be better to define the **name* * of the env vars as const too.